package.json

```diff
@@ -51,12 +58,15 @@
     "node": ">=4.0.0"
   },
   "scripts": {
-    "test": "mocha ./test/node",
+    "test": "./tools/run_tests.sh",
+    "karma-test": "./tools/run_tests.sh karma",
```

Comment:
Let the bike-shedding commence!!!

I personally think we should name these `test-browser` and `test-node`.